frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -326,6 +327,16 @@ class FieldValuesWidgetInner extends Component {
             optionRenderer={optionRenderer}
             checkedColor={checkedColor}
           />
+        ) : isListMode && hasListValues && !multi ? (
+          <SingleSelectListField
+            isDashboardFilter={parameter}
+            placeholder={tokenFieldPlaceholder}
+            value={value.filter(v => v != null)}
+            onChange={onChange}
+            options={options}
+            optionRenderer={optionRenderer}
+            checkedColor={checkedColor}
+          />
```

Comment:
I know ternaries are considered bad practice but I don't know how to survive without them in a JSX expression